### PR TITLE
Fixed Image and data not being hidden on page flip in team.html in Firefox browser

### LINKED ISSUE #457: Image and data not being hidden on page flip in team.html in Firefox browser
## **Describe the bug**
In Firefox browser in `team.html` page on hover of the cards the image and the data is not being hidden as it is being hidden in chrome or edge, refer the screenshots below.

## **Screenshots**
- In Firefox
image
- In Chrome or Edge
image

### PR DESCRIPTION
**Pull request by ishwari20**
# Description

Fixed Image and data not being hidden on page flip in team.html in Firefox browser
Fixes #457

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## UI /UX changes
   Attach gif or screenshot for changes.
   - Before :
   
Screenshot (197)

   - After :
Screenshot (196)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by ravikr126** [comment]
this issue already done by someone .now it so correct in website @ishwari20

**Comment by ishwari20** [comment]
@ravikr126  But I forked the latest version and the issue was still there.

**Comment by ravikr126** [comment]
But I show there is no any error

**Comment by harikesh409** [comment]
> this issue already done by someone .now it so correct in website @ishwari20

The issue is still present for FireFox browser.

File: css/style.css [hunk 1]
```diff
@@ -505,6 +505,15 @@ HEAD .round-2 {
   transform: rotateY(-180deg);
 }
 
+/*Fixes card flip issue in firefox*/
+.flip-card-front, .flip-card-back {
+  position: absolute;
+  width: 100%;
+  height: 100%;
+  backface-visibility: hidden;
+  transform: rotateX(0deg);
```
Comment (harikesh409) [inline css/style.css:514]: Only this line is new so just add this along with the other properties.

File: css/style.css [hunk 2]
```diff
@@ -505,6 +505,15 @@ HEAD .round-2 {
   transform: rotateY(-180deg);
 }
 
+/*Fixes card flip issue in firefox*/
+.flip-card-front, .flip-card-back {
+  position: absolute;
```
Comment (harikesh409) [inline css/style.css:510]: All these properties are already present.

**Review by harikesh409 — CHANGES_REQUESTED** [review]

**Comment by ishwari20** [comment]
@harikesh409 Done the changes. Please review

**Review by harikesh409 — APPROVED** [review]

**Pull request merged**